flow360/cloud/http_util.py

```diff
@@ -23,7 +23,8 @@ def api_key_auth(request):
     key = api_key()
     if not key:
         raise AuthorisationError(
-            f"API key not found for profile={UserConfig.profile}, please set it by commandline: flow360 configure."
+            f"API key not found for profile={UserConfig.profile} in "
+            f"environment={Env.current.name}, please set it by commandline: flow360 configure."
```

Comment:
env is only for developers, so we don't want to show it usual auth error. There should be a message when used in Env.dev.active() and apikey not found:
`apikey not found for env=dev, set it by running flow360 configure with --dev flag`

Reply:
Fixed in 2509716